onnx/checker.py

```diff
@@ -84,11 +84,15 @@ def check_node(node: NodeProto, ctx: C.CheckerContext = DEFAULT_CONTEXT) -> None
     pass
 
 
-@_create_checker(FunctionProto)
 def check_function(
-    graph: FunctionProto, ctx: C.CheckerContext = DEFAULT_CONTEXT
+    function: FunctionProto, ctx: C.CheckerContext = DEFAULT_CONTEXT
 ) -> None:
-    pass
+    if ctx == DEFAULT_CONTEXT:
```

Comment:
And change this to `if cts is None`, and create a new context

Reply:
Updated. PTAL thanks!